wellregistry/registry/tests/test_views.py

```diff
@@ -51,6 +51,15 @@ def test_display_flag_false_monitoring_locations(self):
         self.assertIn(resp.data['results'][0]['site_no'], ['12345678', '44445555'])
         self.assertIn(resp.data['results'][1]['site_no'], ['12345678', '44445555'])
 
+    def test_site_no_autocomplete_monitoring_locations(self):
```

Comment:
You are not testing the admin here, you are testing the rest endpoint so please remove. The test should go in tests/admin. See https://github.com/ACWI-SOGW/well_registry_management/blob/master/wellregistry/registry/tests/admin/test_monitoring_location.py#L145 for an example of how to set up a view test in the admin.

Reply:
fixed